SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -0,0 +1,329 @@
+#!/bin/bash
+
+# Author: Rodrigo Sobral 2024
+# Copyright CERN
+# Here the script is used to create a virtual environment and install the packages from a provided requirements file.
+
+_log () {
+    if [ "$*" == "ERROR:"* ] || [ "$*" == "WARNING:"* ] || [ "${JUPYTER_DOCKER_STACKS_QUIET}" == "" ]; then
+        echo "$@"
+    fi
+}
+
+# if [ -f "/home/$USER/.bashrc" ]; then
+#     source /home/$USER/.bashrc
+# fi
+# CUSTOMENVS_LIMIT=1
+ACCPY_BASE=$ACCPY_PATH/base
+ACCPY_ALL_VERSIONS_STR="?"
+PYTHON_DEFAULT_PATH=$(which python)
```

Comment:
Can you move this var setting of this acc-py section (up) ? And I would add a comment of the sort: "By default, we use the Python installed in the image". Then below you can have the acc-py section that you can start with a comment "acc-py variable initialization"